Flat/src/PandaAnalyzer.cc

```diff
@@ -1019,6 +1018,13 @@ void PandaAnalyzer::Run()
           }
         }
       }
+    }
+
+    if (isData) {
+      // check the json
+      if (!PassGoodLumis(gt->runNumber,gt->lumiNumber))
```

Comment:
Why was this moved down here? It was one of the first checks to avoid spending CPU on data we don't care about.

Reply:
Well, maybe I can change the order, but the point is that we want to do that check regardless of the MCtrigger. We now need to if statements: (1) for the triggers and (2) for the good run list. I do it in that order, although I can do first (2) and then (1). I can change that, I agree there is no point in checking the triggers if the event will be rejected anyway.

Reply:
Yeah two if statements is allright. It's not expensive to check 2 bools, but it is more expensive to check 100s of bools in the tree.

Reply:
Sure, I agree